ansible-scylla-node/tasks/ssl.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: For every node, check if crt file exists
   stat:
-    path: "./ssl/{{ hostvars[item]['inventory_hostname'] }}/{{ hostvars[item]['inventory_hostname'] }}.crt"
+    path: "{{ scylla_ssl.localhost_cert_path }}/ssl/{{ hostvars[item]['inventory_hostname'] }}/{{ hostvars[item]['inventory_hostname'] }}.crt"
```

Comment:
`scylla_ssl.localhost_cert_path` ?
Should it be `localhost_cert_path`?

Reply:
Where does it come anyway?
A little (a lot!) more text in the patch description is expected.

Reply:
Done